content/md/what-is/what-is-an-attribute.md

```diff
@@ -5,63 +5,113 @@ title: What is an **attribute**?
+
+In Akeneo, attributes are gathered into **families**, so all products belonging to the same family share the same attributes. If you are not familiar with the concept of families, check our [What is family?](what-is-a-family.html) article.
+
+You can easily create attribute groups to gather your attributes and ease their enrichment in the product form. Read the following article to get more information about attribute groups: [What is an attribute group?](what-is-an-attribute-group.html).
+
+# Akeneo attribute types
+
+Here are the attribute types available in Akeneo:
+- **Identifier** attribute: a code to identify your product, **this code must be unique**. It can be a SKU, a MPN... **This attribute is mandatory to create products.**
+- **Text** attribute: a single-line text field that can contain **up to 255 characters**, it is usually used for a product name.
+- **Text area** attribute: a multi-line text field that can be used for a product description.
+- **Simple select** attribute: a single-choice list coming with custom options. Only one value can be selected amongst the available options. 
+- **Multi select** attribute: a multi-choice list coming with custom options. More than one value can be selected amongst the available options.
+- **Yes/No** attribute: a boolean attribute
+- **Date** attribute: a date field, the PIM will display a calendar to choose the date, which includes day, month and year.
+- **Number** attribute: a single-line field that can only contain digits. 
+- **Metric** attribute: a single-line field composed of a 1st field containing a number and a second field containing a metric unit. It allows you to automatically convert metric values into others to fit your export needs. You can check the [Manage your Channels](manage-your-channels.html#create-a-channel) article to have more information about this feature.
+- **Price** attribute: a price attribute with values per currency. The displayed values will depend on the currencies enabled in the PIM. Learn [here](manage-your-currencies.html#enabledisable-a-currency) how to enable or disable a currency in Akeneo PIM.
+- **Image** attribute: a drag and drop down zone to upload an image (extensions allowed: gif, jfif, jif, jpeg, jpg, pdf, png, psd, tif, tiff)
+- **file** attribute: a drag and drop down zone to upload a file (extensions allowed: csv, doc, docx, mp3, pdf)
+- **Asset collection** attribute **_(Entreprise Edition only)_**: an advanced attribute type to manage several digital resources like a video, picture, pdf file... Read the following article [What is an asset?](what-is-an-asset.html) to learn more about this attribute type
+- **Reference entity simple link** attribute **_(Entreprise Edition only)_**: allows enriching common data related to products with a rich content (text, images...), more complex than just a code and labels. 
```

Comment:
Tu peux peut-être ajouté le comme pour les assets, le lien vers "What is a reference entity?"

Reply:
```suggestion
- **Reference entity simple link** attribute **_(Entreprise Edition only)_**: allows enriching common data related to products with a rich content (text, images...), more complex than just a code and labels. Check our dedicated article [What is a reference entity?](what-about-reference-entities.html) to learn more about this attribute type.
```